### ISSUE TITLE
Connect input descriptions to input fields with aria-describedby

### ISSUE THREAD

**Issue by Veyfeyken** [a-ui/core_branding_scss#167]
[atoms/atoms.input](https://a-ui.github.io/core_branding_scss/#atoms-input)

The following recommendations are accessibility improvements but not required to achieve WCAG compliance.

* Placing the input description between the label and input is more user friendly. 
* You can connect the description to the input with an aria-describedby attribute. This will make screenreaders read the description together with the label when the input receives focus. This is especially useful for screenreaders users in forms mode. Without this connection, only the labels of inputs will be read out loud in forms mode. Screenreader users will miss important information that can help to fill out the form correctly.
* Delete placeholder attributes everywhere. They add no benefit, only confusion. Usability testing indicates it's better to leave input fields empty.

Current:

```html
<div class="a-input">
  <label class="a-input__label" for="text-field">Text field</label>
  <input type="text" name="text-field" id="text-field" placeholder="Placeholder" />
  <small>Input description</small>
</div>
```

![screenshot input](https://user-images.githubusercontent.com/870049/98800822-378d8e80-2411-11eb-95a1-4fcdf301d705.jpg)

Suggestion:

```html
<div class="a-input">
  <label class="a-input__label" for="text-field">Text field</label>
  <small id="text-field-hint">Input description</small>
  <input type="text" name="text-field" id="text-field" aria-describedby="text-field-hint"/>
</div>
```

![screenshot input with description between label and input](https://user-images.githubusercontent.com/870049/98801547-3ad54a00-2412-11eb-94e5-a555b800fc0a.png)

### Links and articles

* [Gov.uk Hint Text Example](https://design-system.service.gov.uk/components/text-input/input-hint-text/index.html)
* [Nielsen Norman Group: Placeholders in Form Fields Are Harmful](https://www.nngroup.com/articles/form-design-placeholders/)

**Comment by TriangleJuice** [comment]
Hey @Veyfeyken this issue should be fixed in the next iteration of our core branding (`v4.3.0`)
